runner.go

```diff
@@ -152,6 +152,9 @@ func (r *runner) constructCommand(projectDir string) string {
 	// We start code-server such that extensions installed through the UI are placed in the host's extension dir.
 	cmd := fmt.Sprintf(`set -euxo pipefail || exit 1
 cd %v
+# This is necessary in case the .vscode directory wasn't created inside the container, as mounting to the host
+# extension dir will create it as root.
+sudo chown user:user ~/.vscode
```

Comment:
Do we need to run `chown -R`?

Reply:
Mm we do.

Reply:
https://github.com/cdr/sail/commit/ca753921313c899a87d8cbda17b18c94cebf180c

Reply:
Sorry about that, saw it after I approved